java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -4342,37 +4342,78 @@ void testMakeStruct() {
 
   @Test
   void testMakeListEmpty() {
-    final int numRows = 10;
-    try (ColumnVector expected =
+
+    final int numRows = 4;
+    List<List<Integer>> mainList1 = new ArrayList<>();
```

Comment:
The code below is expecting a list-of-list-of-string, but this is constructing a list-of-list-of-integer.  Ends up not mattering in practice, but it is confusing.

Reply:
Oops, Good findings, When constructing List of List type, There are no real values for the child data column, only offset is valid, So List<List<Integer>> will not cause any issue. But I will change it to List<List< String>>